description: avoid third party jinja and markupsafe to pick up system versions

Index: sid/third_party/WebKit/Source/build/scripts/scripts.gypi
===================================================================
--- sid.orig/third_party/WebKit/Source/build/scripts/scripts.gypi
+++ sid/third_party/WebKit/Source/build/scripts/scripts.gypi
@@ -4,8 +4,6 @@
         'scripts_for_in_files': [
             # jinja2/__init__.py contains version string, so sufficient as
             # dependency for whole jinja2 package
-            '<(DEPTH)/third_party/jinja2/__init__.py',
-            '<(DEPTH)/third_party/markupsafe/__init__.py',  # jinja2 dep
             'hasher.py',
             'in_file.py',
             'in_generator.py',
